example/lib/main.dart

```diff
@@ -52,6 +52,7 @@ class _MyHomePageState extends State<MyHomePage> with TickerProviderStateMixin {
     final _selectedDay = DateTime.now();
 
     _events = {
+      DateTime(2019, 1, 1): ['Event on New Year\s'],
```

Comment:
I know this is just for testing the API changes, but please remove it afterwards